ClientManagementService/ClientManagementService.Test/Service/ClientServiceTest.cs

```diff
@@ -526,6 +528,7 @@ public void ClientLogIn_PasswordDoesNotMatch()
             var clientService = new ClientService(_clientRepository.Object, _passwordService);
 
             Assert.ThrowsAsync<ArgumentException>(() => clientService.ClientLogin("jdoe", "Test123!"));
+            _clientRepository.Verify(c => c.IncrementClientFailedLoginAttempts(It.Is<long>(id => id == 1)), Times.Once);

```

Comment:
there is one more thing you have to verify. trace the client login code and see what gets called when increment failed login attempt is invoked.

Reply:
updated